src/Raku/ast/signature.rakumod

```diff
@@ -1252,7 +1252,8 @@ class RakuAST::Parameter
                           QAST::Op.new(:op<iseq_s>, $wval, $temp-qast-var));
         }
 
-        $context.ensure-sc(nqp::getattr($param-obj, Parameter, '$!container_descriptor'));
+        my $container_descriptor := $param-obj.container_descriptor;
+        $context.ensure-sc($container_descriptor) if $container_descriptor;
```

Comment:
If the object is nqp::null, then a better fix would be `unless nqp::isnull($container_descriptor)`

Reply:
Yeah, the problem was a nqp::null. My thought was that testing for truthiness would be good enough and it would match with the other test in line 1299. But your suggestion is more precise, I've pushed a second commit.